### PR TITLE
docs(VirtualizedGrid): should use Text instead of inline string

### LINKED ISSUE #2505: Change `VirtualizedGrid` catalog component to use `Text` component
In the file of:
`packages/core/src/components/VirtualizedGrid/__stories__/VirtualizedGrid.stories.helpers.js`
which can be seen on our [Catalog](https://vibe.monday.com/?path=/docs/catalog--docs) page, we do not currently use Vibe's `Text` component, but instead use inline string.
It also causes theme issues where a story doesn't look right on dark themes.

We need to change the wrapper `div` to be `Text` instead.
```typescript
<div
  style={{
    backgroundColor,
    height: item.height,
    width: item.width,
    display: "flex",
    alignItems: "center",
    justifyContent: "center"
  }}
>
  {item.value}
</div>
```
--> Something like
```typescript
<Text
  color={Text.colors.FIXED_DARK}
  style={{
    backgroundColor,
    height: item.height,
    width: item.width,
    display: "flex",
    alignItems: "center",
    justifyContent: "center"
  }}
>
  {item.value}
</Text>
```

Notice the `FIXED_DARK` color addition as well - we need it so it would be shown correct on all themes.

monday ticket:
https://monday.monday.com/boards/3532714909/pulses/7080412658

### PR DESCRIPTION
**Pull request by GurneeshBudhiraja**
Changes made:
Replaced the `div` with the `Text` component in the `packages/core/src/components/VirtualizedGrid/__stories__/VirtualizedGrid.stories.helpers.js` file.

- [X] I have read the [Contribution Guide](../packages/core/CONTRIBUTING.md) for this project.

Resolves #2505

### CODE REVIEW

**Review by YossiSaadi — DISMISSED** [review]
Awesome! thanks @GurneeshBudhiraja

File: packages/core/src/components/VirtualizedGrid/__stories__/VirtualizedGrid.stories.helpers.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import { Text } from "src/components/Text";
```
Comment (YossiSaadi) [inline packages/core/src/components/VirtualizedGrid/__stories__/VirtualizedGrid.stories.helpers.js:2]: just noticed this is being imported wrongly
did you ran Storybook to make sure it works?
the import should look something like
```
import { Text } from "../../Text";
```
as we do not have aliases yet in the project

**Review by YossiSaadi — CHANGES_REQUESTED** [review]
Can you please fix my comment @GurneeshBudhiraja ?

File: packages/core/src/components/VirtualizedGrid/__stories__/VirtualizedGrid.stories.helpers.js (hunk 1 above)
Comment (GurneeshBudhiraja) [reply packages/core/src/components/VirtualizedGrid/__stories__/VirtualizedGrid.stories.helpers.js]: My bad.
Just fixed the import statement. Also ran storybook and tests to make sure everything is alright.

**Review by YossiSaadi — APPROVED** [review]
💯

**Pull request closed without merging**